taca/cli.py

```diff
@@ -0,0 +1,32 @@
+import os
```

Comment:
I don't know if you worry about encoding? I'm just obsessive about putting it on top of every Python file I write.

``# -*- coding: utf-8 -*-``

https://www.python.org/dev/peps/pep-0263/

Have a Sublime snippet free of charge:
```xml
<snippet>
  <content><![CDATA[
# -*- coding: utf-8 -*-
]]></content>
  <!-- Optional: Set a tabTrigger to define how to trigger the snippet -->
  <tabTrigger>enc</tabTrigger>
  <!-- Optional: Set a scope to limit where the snippet will trigger -->
  <scope>source.python</scope>
</snippet>
```